### PR TITLE
Make viash automount prefix configurable

### PR DESCRIPTION
**Pull request by rcannood**
### MINOR CHANGES

* `ExecutableRunner`: Add parameter `docker_automount_prefix` to allow for a custom prefix for automounted folders (PR #xxx).

## Related issue(s)

Closes #xxxx 

## Type of Change

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] New functionality (non-breaking change which adds functionality)
- [ ] Major change (non-breaking change which modifies existing functionality)
- [x] Minor change (non-breaking change which does not modify existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] This change requires a documentation update

## Checklist

Requirements:

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc.
- [x] I have performed a self-review of my code by checking the "Changed Files" tab.
- [x] My code follows the code style of this project.

Tests:

- [x] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.

Documentation:

- [x] Proposed changes are described in the CHANGELOG.md.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: src/main/scala/io/viash/runners/ExecutableRunner.scala [hunk 1]
```diff
@@ -101,6 +101,11 @@ final case class ExecutableRunner(
   @default("Empty")
   docker_run_args: OneOrMore[String] = Nil,
 
+  @description("The default prefix to use for automounting directories in Docker. This is used to mount directories from the host into the Docker container.")
+  @example("docker_automount_prefix: /viash_automount", "yaml")
+  @default("/viash_automount")
```
Comment (Grifs) [inline src/main/scala/io/viash/runners/ExecutableRunner.scala:106]:
```suggestion
  @default("/viash_automount")
  @since("since 0.9.0")
```

**Review by Grifs — COMMENTED** [review]
would it make sense to use a `---docker_automount_prefix` argument instead of an environment variable?

Pro: 
- Cleaner, more defined
- `--help` could output these ... once we actually implement this feature

Con:
- No straight forward way to apply this globally in e.g. `.env`; each component needs to have it applied explicitly

**Comment by rcannood** [comment]
Interesting thought! I agree with the pros and cons :P

:thinking: :gear:

**Comment by rcannood** [comment]
To clarify, I agree with the pros and cons. In a use case where the viash automount actually needs to be configurable, it would be useful to not have to pass it as a flag every time.

For that reason I would be inclined to either only support the env variable, or support both the env and the flag.

If the `---flags` ever end up in the `--help`, this could even be documented as such.

**Review by rcannood — COMMENTED** [review]
Something like this? @Grifs

**Comment by Grifs** [comment]
for future reference, we decided to keep the functionality but make it slightly more 'hidden'. The folder path is no longer hard coded but instead allows an environment variable to be set. If no value is defined, there is no functional change. However, the additional code to set it in the executable runner is removed for now.

**Review by Grifs — APPROVED** [review]

**Pull request merged**